src/app.js

```diff
@@ -462,6 +462,7 @@ var run = function () {
       $('#ghostbar').remove()
       $(document).unbind('mousemove')
       dragging = false
+      delta = (delta < 50) ? 50 : delta
```

Comment:
Actually for me, the editor does not shrink arbitrarily anyway. Can we somehow find out what the correct bound is?